en/index.md

```diff
@@ -524,6 +524,46 @@ For example:
 
 - **Don’t use:** Click the ![pencil](https://assets.ubuntu.com/v1/633aab2b-pencil.png) pencil icon to edit the post.
 
+## Hyperlinks
+
+Here are some pointers about the general usage of hyperlinks and how to format them correctly.
+
+### General usage
+Avoid excessive links in the same paragraph or instruction. If you find yourself introducing several links in your content, consider listing them in a separate section called "Related topics", "Additional resources", or similar.
+
+When possible, avoid external links. They can easily become outdated, making the document harder to maintain.
+
+When linking to versioned files or specific lines of code, copy the **permalink** instead of the URL. This will ensure the link is bound to the current revision of the file, so it will direct to the same content even if the file changes.
+
+### Formatting
+Make the link text match the exact title or heading that you are referencing. Avoid phrases like "this document", "this article", "click here".
+
+If linking to a section called "Formatting":
+
+- Use: `[Formatting](#formatting)` 
+- Don't use: `[Guidelines to format hyperlinks](#formatting)`
+- Don't use: `[this section](#formatting)`
+
+In a full sentence, introduce the link with "For more information, see ..." or similar. 
+
+- Use: `For more information about formatting hyperlinks, see [Formatting](#formatting)`
+- Don't use: `See [Formatting](#formatting).`
+
+Avoid using a URL as the linked text.
+- Use: `[Page title](https://page-url.com)`
+- Don't use: `[https://page-url.com](https://page-url.com)`
+
+For external links, inform the user the link is external to the docs by specifying the source. This can either be done as part of the sentence, or with a delimiter.
+
+- Use: `For more information, see the ExternalStyleGuide page [How to format hyperlinks](www.external-style-guide.com/hyperlinks).`
+- Use: `- ExternalStyleGuide | How to format hyperlinks](www.external-style-guide.com/hyperlinks)`
```

Comment:
I think this is missing a "[".

Reply:
Probably instead of the leading hyphen, or has that some meaning?

Reply:
- The missing `[` is indeed a typo
- The leading hyphen is meant to denote a bullet point in markdown, as I wanted to show a use case for using a short-hand hyperlink that specifies the source. Definitely see how that is not quite clear here.
- Using a pipe as a delimiter between the external source name and the link itself is totally up for debate. Sometimes it looks nice, other times it can look a bit confusing...

I'm partial now to omitting this example and the delimiter suggestion entirely.